contracts/src/L2/predeploys/L1GasPriceOracle.sol

```diff
@@ -48,6 +86,27 @@ contract L1GasPriceOracle is OwnableBase, IL1GasPriceOracle {
     /// @notice The address of whitelist contract.
     IWhitelist public whitelist;
 
+    /// @inheritdoc IL1GasPriceOracle
+    uint256 public override l1BlobBaseFee;
+
+    /// @inheritdoc IL1GasPriceOracle
+    uint256 public override commitScalar;
+
+    /// @inheritdoc IL1GasPriceOracle
+    uint256 public override blobScalar;
+
+    /// @notice Indicates whether the network has gone through the Curie upgrade.
+    bool public isCurie;
```

Comment:
So need to set this slot to `true` in the hardfork block?

Reply:
Set here: https://github.com/scroll-tech/go-ethereum/pull/755/files#diff-bff5ba79b7b5a8983f6156c393a80a9f3283f7692ff6f641957b5e8b5752d46aR19